forge/routes/api/shared/users.js

```diff
@@ -112,7 +119,20 @@ module.exports = {
             }
             await user.save()
 
-            // re-send verification email if a user was previously verifed and is now not verified
+            if (user.username !== originalUser.username) {
+                await app.postoffice.send(originalUser, 'UsernameChanged', {
+                    oldUsername: originalUser.username,
+                    newUsername: user.username
+                })
+            }
+            if (user.email !== originalUser.email) {
```

Comment:
@Steve-Mcl @knolleary The template in `forge/postoffice/templates/PasswordChanged.js` is never used?

Reply:
Ignore me, there's another file below.

Reply:
It's used here: https://github.com/flowforge/flowforge/blob/main/forge/routes/api/user.js#L55